charts/lighthouse/.helmignore

```diff
@@ -19,3 +19,6 @@
 .project
 .idea/
 *.tmproj
+# Project files
+Makefile
+OWNERS
```

Comment:
why adding OWNERS? There is none right?

Reply:
true, there's no OWNERS file. i added it because it's so common. i can remove it if you prefer.